index.bs

```diff
@@ -2043,18 +2044,32 @@ is a request, rather than a guarantee.
 
 <h3 id="string-constants">Use strings for constants and enums</h3>
 
-If your API needs a constant, or a set of enumerated values,
-use string values.
+Use strings as the values for constants or sets of enumerated values.
 
-Strings are easier for developers to inspect,
-and in JavaScript engines there is no performance benefit
-from using integers instead of strings.
+Strings are easier for developers to inspect values
+and read code that uses those values.
+In JavaScript engines there is no performance benefit
+to using integers instead of strings.
+The [WebIDL enum type](https://webidl.spec.whatwg.org/#idl-enums)
+only permits the use of string values.
```

Comment:
This could be more attached to the rest of the paragraph ... maybe

```suggestion
The values of a [WebIDL enum type](https://webidl.spec.whatwg.org/#idl-enums)
are strings in order to follow this principle.
```